server/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/ConcurrentRebalanceAllocationDecider.java

```diff
@@ -62,10 +62,17 @@ public Decision canRebalance(ShardRouting shardRouting, RoutingAllocation alloca
 
     @Override
     public Decision canRebalance(RoutingAllocation allocation) {
+        int relocatingShards = allocation.routingNodes().getRelocatingShardCount();
+        if (allocation.isSimulating() && relocatingShards >= 1) {
```

Comment:
Maybe we should use `2` here just for consistency with the past default behaviour? I think I've seen odd things happening at `1` sometimes.

Reply:
Is there a chance you could describe or link some of them? 
May be we could add some tests for them and assess the better value